gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/permissions/EnvironmentPermission.java

```diff
@@ -47,7 +47,8 @@ public enum EnvironmentPermission implements Permission {
     CLIENT_REGISTRATION_PROVIDER("CLIENT_REGISTRATION_PROVIDER", 3500),
     THEME("THEME", 3600),
     IDENTITY_PROVIDER_ACTIVATION("IDENTITY_PROVIDER_ACTIVATION", 3700),
-    INTEGRATION("INTEGRATION", 3800);
+    INTEGRATION("INTEGRATION", 3800),
+    FLOWS("FLOWS", 3900);
```

Comment:
as discussed, we will wait for the feedback from Anthony about naming

Reply:
Sure :) I also prefer to keep ENVIRONMENT_FLOWS here too